build-system/global-configs/canary-config.json

```diff
@@ -8,7 +8,6 @@
   "expDoubleclickA4A": 1,
   "pan-y": 1,
   "a4aProfilingRate": 1,
-  "amp-form-custom-validations": 1,
```

Comment:
I believe this should be removed afterwards.  the actual experiment check can only make to canary this week, while the config will be live in prod.